src/main/java/org/openjdk/jextract/impl/UnsupportedFilter.java

```diff
@@ -105,21 +97,20 @@ public Void visitFunction(Function funcTree, Declaration parent) {
 
     @Override
     public Void visitVariable(Variable varTree, Declaration parent) {
-        String unsupportedType = firstUnsupportedType(varTree.type());
+        Type type = varTree.type();
```

Comment:
A bit of a snag with the IR - in a lot of situations we have to recursively visit the declaration corresponding to a declared type (if any is found), as that could be some anonymous struct/union.